service/service.go

```diff
@@ -388,6 +390,13 @@ func (s *service) BeforeServe(
 	if sdcPrefix, ok := csictx.LookupEnv(ctx, EnvSDCPrefix); ok {
 		opts.SdcPrefix = sdcPrefix
 	}
+	if approveSDC, ok := csictx.LookupEnv(ctx, EnvIsApproveSDCEnabled); ok {
```

Comment:
Could simplify code, set opts.IsApproveSDCEnabled set to false by default and remove else clause. Upto you.